src/Builder.fs

```diff
@@ -27,9 +27,9 @@ type RequestBuilder () =
         |> Seq.map func
         |> sequential
 
-    member _.Bind(source: HttpHandler<'a, 'b, 'r, 'err>, fn: 'b -> HttpHandler<'a, 'c, 'r, 'err>) : HttpHandler<'a, 'c, 'r, 'err> =
+    member _.Bind(source: HttpHandler<'T, 'T2, 'TResult, 'TError>, fn: 'T2 -> HttpHandler<'T, 'T3, 'TResult, 'TError>) : HttpHandler<'T, 'T3, 'TResult, 'TError> =
```

Comment:
I think it would be a little bit better to call `T2` `TNext` as in `HttpHandler<'T, 'TNext, 'TResult, 'TError>`

Reply:
But there is a T3 as well. TNext1, TNext2?

Reply:
Aha, I didn't notice. `TNext1`, `TNext2` would be the best, IMHO.